samples/client/petstore/rust/src/apis/get_pet_by_id_api.rs

```diff
@@ -0,0 +1,23 @@
+use hyper;
+use serde_json;
+use futures::{Future, Stream};
+
+use super::models;
+use super::Error;
+
+pub fn get_pet_by_id<C: hyper::client::Connect>(
+    prefix: &str,
+    cli: &hyper::client::Client<C>,
+    pet_id: i64,
+    ) -> Box<Future<Item = models::Pet, Error = Error>> {
```

Comment:
It's also worth sharing other API clients usually come with a corresponding "Async" method (e.g. [addPetAsync](https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/csharp/SwaggerClient/src/IO.Swagger/Api/PetApi.cs#L572)) for async function call. Not sure how that should be done in Rust but worth thinking ahead with this future requirement.